GiniVision/Classes/Core/AnalysisViewController.swift

```diff
@@ -135,7 +135,7 @@ import UIKit
         super.loadView()
         imageView.image = document.previewImage
         edgesForExtendedLayout = []
-        view.backgroundColor = .black
+        view.backgroundColor = GiniConfiguration.shared.backgroundColor
```

Comment:
I'm a bit confused. There are some view controllers where the configuration is injected in the constructor and here, the shared one is used. Is there a reason for this? If not, I'd say that it would have been better to be consistent and inject it here too.

Reply:
I saw this in the AnalysisViewController, but I was thinking on creating a different PR for this. Maybe it makes sense to add it in this one 👍